src/styles/main.scss

```diff
@@ -1,3 +1,77 @@
+$calendar-font: Arial, Helvetica, sans-serif;
+$calendar-padding: 10px;
+$square-size: 100px;
+$square-gap-size: 1px;
+$square-size-full: calc($square-size + $square-gap-size);
+$square-border-size: 1px;
+$square-main-color: #eee;
+$square-hover-color: pink;
```

Comment:
You could maybe rename square to like day square but it's also easy to understand with just square